backend/src/sync/client.rs

```diff
@@ -177,6 +177,21 @@ impl OcClient {
         self.http_client.request(req).await.map_err(Into::into)
     }
 
+    pub async fn has_active_workflows(&self, oc_id: &String) -> Result<bool> {
+        let pq = format!("/workflow/mediaPackage/{oc_id}/hasActiveWorkflows");
+        let req = self.authed_req_builder(&self.external_api_node, &pq)
```

Comment:
Mh so, this is not using the "external API" anymore (those paths would start with `/api`). So not sure if it's fine to always use the `external_api_node`. I just asked in the community chat and hopefully it is fine! Otherwise we would need to add another node config or.. sth.

Edit: Ok so it seems like this API is always available on the admin node. So we can't really use `external_api_node` I am afraid :/ We also don't have a fitting node already, so I fear we must add a new node config. Not sure if we should call it `admin_node` or `workflow_api_node` or sth... mh.

The other thing is that the stability of this API is not guaranteed too much. I'm sure lots of other apps use it, but maybe Tobira should be able to deal with the case that the API doesn't exist? Maybe in that case just don't block editing ACL? Mhhh